Updated "About us" section

Gumbosh opened the pull request:
- Add images for "About us" section.
- Updated "About us" section for desktop/tablet.

marcinstenka committed:
feat: added mobile menu initial structure and styles

marcinstenka committed:
feat: mobile menu disappears after clicking link

marcinstenka committed:
style: bg & links colors changed

marcinstenka committed:
fix: width&height of menu changed from screen to full

Gumbosh committed:
Add images for "About us" section

Gumbosh committed:
Update "About us" section for desktop/tablet

marcinstenka committed:
Merge pull request #8 from AleksandraWojcikk/mobile-menu

Mobile menu

marcinstenka committed:
Merge branch 'about-us' of https://github.com/AleksandraWojcikk/Beauty-Salon-Landing-Page into about-us

marcinstenka committed:
style: responsive on mobile, changed image displaying

marcinstenka commented:
I added some responsive styling and changed how images are displayed. What do you think>

Gumbosh commented:
> I added some responsive styling and changed how images are displayed. What do you think>

Looks good, one thing that i would keep is section height at 100vh.

Gumbosh committed:
Changed section height to 100vh
